readme.md

```diff
@@ -39,6 +39,13 @@ chrome_options.add_argument("--proxy-server={0}".format(proxy.proxy))
 browser = webdriver.Chrome(chrome_options = chrome_options)
 ```
 
+Before Running Tests
```

Comment:
This part shouldnt be required, in https://github.com/AutomatedTester/browsermob-proxy-py/blob/master/browsermobproxy/server.py#L53  the code should start up the process.